test/action.yaml

```diff
@@ -59,6 +53,6 @@ runs:
         AWS_SECRET_ACCESS_KEY: ${{ inputs.aws-secret-access-key }}
         TF_CLI_CONFIG_FILE: ${{ inputs.terraform-cli-config-file }}
     - name: Post run
-      if: always() && env._ACTIONS_TF_TEST_CREDENTIALS
+      if: always() && $ {{ steps.create-credentials-file.outputs.terraform-cli-config-file-created }}
```

Comment:
Oof I forgot about this... :/ 

Stupid composites not allowing post-run hooks. This being super-specific to the auth action makes me think it should have a `actions-tf/auth/post-run` or `actions-tf/auth/cleanup` that wraps it and takes no inputs (e.g. instead it reads the `${{ env... }}`? Dunno if that makes sense.

Reply:
Going to leave this in place for now as we discussed.